conjure-scala-jaxrs-client/src/main/java/com/palantir/conjure/java/serialization/ScalaObjectMappers.java

```diff
@@ -19,38 +19,65 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.introspect.AnnotationIntrospectorPair;
 import com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.dataformat.cbor.databind.CBORMapper;
+import com.fasterxml.jackson.dataformat.smile.databind.SmileMapper;
 import com.fasterxml.jackson.module.scala.DefaultScalaModule;
 import com.fasterxml.jackson.module.scala.introspect.ScalaAnnotationIntrospector$;
 
 public final class ScalaObjectMappers {
 
     private ScalaObjectMappers() {}
 
+    public static JsonMapper newClientJsonMapper() {
+        return withScalaSupport(ObjectMappers.newClientJsonMapper());
+    }
+
+    public static CBORMapper newClientCborMapper() {
+        return withScalaSupport(ObjectMappers.newClientCborMapper());
+    }
+
+    public static SmileMapper newClientSmileMapper() {
+        return withScalaSupport(ObjectMappers.newClientSmileMapper());
+    }
+
+    public static JsonMapper newServerJsonMapper() {
+        return withScalaSupport(ObjectMappers.newServerJsonMapper());
+    }
+
+    public static CBORMapper newServerCborMapper() {
+        return withScalaSupport(ObjectMappers.newServerCborMapper());
+    }
+
+    public static SmileMapper newServerSmileMapper() {
+        return withScalaSupport(ObjectMappers.newServerSmileMapper());
```

Comment:
no scala on new apis please

Reply:
This is used here:
https://github.com/palantir/conjure-java-runtime/blob/e10e3578a38cb07310ddbb31b6416f1e6dd811de/conjure-scala-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/FeignJaxRsScalaClientBuilder.java#L27-L28

I originally changed the return type of the `AbstractFeignJaxRsClientBuilder` methods since they were package private. But given that someone could override this, I decided not to.

Would you prefer to override these methods to use the more specific types wherever possible? Or just change the `FeignJaxRsScalaClientBuilder` constants to `ObjectMapper`.

Reply:
This is also used in the Witchcraft PR that follows from this PR